### PR TITLE
Not depending on non-json sources to generate html. removed saveJson option

### PR DESCRIPTION
**Pull request by SirLenz0rlot**
As described here: https://github.com/mavdi/grunt-cucumberjs/issues/71.

Sorry for the cluttering, I had to pick up the changes from @gkushang from PR #75.
Feel free to cherry pick my last commits (https://github.com/mavdi/grunt-cucumberjs/pull/76/commits/3aff0963f999c866b59d3bb18c95c6afb92e034e and https://github.com/mavdi/grunt-cucumberjs/pull/76/commits/e22b0e097265d115c324009ec82ce1349d9cfea6)

### CODE REVIEW

**Comment by gkushang** [comment]
@SirLenz0rlot can you pull develop from upstream?

File: tasks/cucumber.js [hunk 1]
```diff
@@ -78,9 +73,9 @@ module.exports = function(grunt) {
                     options.isHtml = true;
                     commands.push('-f', 'json:' + options.output + '.json');
                 } else {
-                    commands.push('-f', format);
+                    commands.push('-f', options.format);
```
Comment (gkushang) [inline tasks/cucumber.js]: can you revert this change? This is a support for multi formatter

File: lib/processHandler.js [hunk 2]
```diff
@@ -80,37 +81,31 @@ module.exports = function HandleUsingProcess(grunt, options, commands, callback)
...
             } else {
                 var jsonOutputFilePath = options.output + '.json';
-                var jsonOutput;
+
 
                 if (fs.existsSync(options.output + '.json')) {
                     jsonOutput= JSON.stringify(jsonFile.readFileSync(jsonOutputFilePath));
                 } else {
-                    jsonOutput = Buffer.concat(buffer).toString();
+                    grunt.log.error('Unable to find cucumberjs json output file.');
```
Comment (gkushang) [inline lib/processHandler.js]: let's log more specific error message `grunt.log.error('Unable to find cucumberjs json output file \'%s\'', jsonOutputFilePath);`

**Comment by SirLenz0rlot** [comment]
@gkushang `develop` isn't been updated for a year.
Pulled `master` instead, making this PR a lot cleaner :)

**Comment by SirLenz0rlot** [comment]
@gkushang could you push a new version to npm once merged?
